app/helpers/physical_server_helper/textual_summary.rb

```diff
@@ -1,10 +1,16 @@
 module PhysicalServerHelper::TextualSummary
   def textual_group_properties
-    %i(name model product_name manufacturer machine_type serial_number ems_ref)
+    TextualGroup.new(
+      _("Properties"),
+      %i(name model product_name manufacturer machine_type serial_number ems_ref)
+    )
   end
 
   def textual_group_relationships
-    %i(host)
+    TextualGroup.new(
+      _("Relationships"),
+      %i(host)
```

Comment:
Should it point to a particular host? If so, you need to provide a host id in that link.

I noticed one more thing, the `Relationships` in the `listnav` PR 
https://github.com/ManageIQ/manageiq-ui-classic/pull/1195 
- looks for `@record.host` first before showing that Relationship
- shows the Parent Provider (`@record.ext_management_system`), which I think is a good idea. 
Could you add that relationship here as well?